app/controllers/users_controller.erb

```diff
@@ -0,0 +1,27 @@
+class UserController < ApplicationController
+
+    def new
+        @user = User.new
+    end
+
+    def create
+        @user = User.new(user_params)
+        if @user.save
+            session[:user_id] = @user.id

```

Comment:
should we be concerned about this being a nil value?

Reply:
def create
  @user = User.new(user_params)
  if @user.save
    session[:user_id] = @user.id
  else
    render :new
  end
end